src/olympia/migrations/938-drop-l10n-from-bio-user-profile.sql

```diff
@@ -0,0 +1,23 @@
+/* FOREIGN KEY is prod's - you will need to change this for your local DB */
+
+/* Add the new column and drop the constraint so we can clear the translations.*/
+ALTER TABLE `users`
+    ADD COLUMN `_bio` longtext NULL,
+    DROP FOREIGN KEY `users_ibfk_1`;
+
+/* Copy across the translated values.  There are some users that have multiple
+localizations so we're going to use the default locale (lang) localization. */
+UPDATE `users`
+    SET `_bio` = (
+        SELECT `localized_string` FROM `translations`
+        WHERE `id`=`bio` AND `locale`=`lang`);
+
+/* Clear out the now obsolete translations. */
+DELETE FROM `translations` WHERE `id` in (SELECT `bio` FROM `users`);
+
+/* Drop the old columns and rename bio_ to take the place of bio. */
+ALTER TABLE `users`
+    DROP COLUMN `bio`,
+    DROP COLUMN lang,
+    DROP COLUMN region,
+    CHANGE COLUMN `_bio` `bio` longtext NULL;
```

Comment:
I actually agree with @diox here, we shouldn't rename the columns. I wonder if we could just do a staged-deploy (like, 2-3 deploys after another) instead of waiting for another week. As in, create `2017.05.04-stage1`, `2017.05.04-stage2`, `2017.05.04-stage3` tags with the appropriate migrations added so that we avoid any potential downtime.

Imho it'd be best to change the model to refer to `_bio` and avoid the downtime between the column-drop and the rename. Ops afair has tools to create synced tables so that we can transparently move between schemas (**pt-online-schema-change**) and we should make use of that, give it more tries here and there even if it's more work but we gain experience in these kinds of deployments like that.